CommunityToolkit.Net.Authentication.Msal/MsalProvider.cs

```diff
@@ -6,102 +6,70 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Net.Http;
+using System.Net.Http.Headers;
 using System.Reflection;
 using System.Threading.Tasks;
-using Microsoft.Graph;
 using Microsoft.Identity.Client;
-using Microsoft.Toolkit.Graph.Extensions;
 
-namespace Microsoft.Toolkit.Graph.Providers
+namespace CommunityToolkit.Net.Authentication.Msal
```

Comment:
We could keep this in the flat `CommunityToolkit.Net.Authentication` namespace, so you'll just see all the providers available once you add the package. That way you don't have a ton of extra using statements everywhere. We've been doing this flattening a bit for 7.0 as well. Thoughts?

Reply:
Love it, I'll simplify the namespace